schema/base/landuse.yaml

```diff
@@ -0,0 +1,159 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Landuse Schema
+description: Landuse features from OpenStreetMap
+type: object
+properties:
+  geometry:
+    description: Landuse features from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Point.json
+      - "$ref": https://geojson.org/schema/LineString.json
+      - "$ref": https://geojson.org/schema/MultiLineString.json
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ./defs.yaml#/$defs/propertyContainers/osmPropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - class
+      - subclass
+    properties:
+      class:
+        description: Broad type of land
+        type: string
+        enum:
+        - agriculture
+        - airport
+        - aquaculture
+        - campground
+        - cemetary
+        - conservation
+        - construction
+        - developed
+        - education
+        - entertainment
+        - golf
+        - horticulture
+        - landfill
+        - medical
+        - military
+        - park
+        - public
+        - protected
+        - recreation
+        - religious
+        - residential
+        - resource_extraction

```

Comment:
The majority of these values are all pass through from OSM `landuse`= or `natural`= tags, which are snake cased. I think there is value in leaving them untouched from the source, but I could be convinced otherwise.

Reply:
The schema standard is camel case. We're also already translating all the snake case things like this to camel case in transportation. This introduces a conflict and inconsistency....

Ideally the schema will present to Overture consumers as one consistent whole that's easy to understand, so IMO we need to resolve the conflict and inconsistency....

Reply:
fair